vuepress/docs/next/tutorials/devops/extend-plugin-descriptor.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebarDepth: 2
+---
+
+# Leverage Plugin Environment Variables 
+
+This tutorial describes how to use environment variables and secrets in the plugin descriptor  of an Entando Component Generator built project to customize microservices behavior. You can use the same docker image to run during development as in production by using the environment variables as a configuration mechanism for your pods or for passing sensitive information. A new validation step is embedded in the Entando process to manage possible incompatible properties.
+
+
+## Entando Bundle Plugin Descriptor
+Currently, environment variables or secrets are not required for the container plugin descriptor file. The plugin descriptor v4 on the Entando Platform adds support for inserting environment variables as allowed by the descriptor.yaml file in k8s. There are two ways to accomplish this. You can deploy a plugin and inject the variables by using a key-value or by reading the value with an existing secret. The bundle is able to receive the configuration values exposed as plain text on the resources/yamls or hidden in the secret. Secrets can be referenced by using a special syntax. This process does not work with Entando plugin descriptor v3 or lower.
+
+### Deploy an EntandoApp with a new secret constraints: ??? not sure what this step is
+
+`name: env-2-secret`\
+`key: env-2secret-key`\
+`value: env2value`
+
+### Add Environment Variables to the Plugin Descriptor file
+
+1. [Create a project bundle](../create/ms/generate-microservices-and-micro-frontends.md) using the latest Entando Blueprint.
+2. Ensure that a Jhipster 7.2 project was created with the command `ent check-env develop`. 
```

Comment:
This step seems redundant. The project would created in step 1 - and this command doesn't generate anything anyway, it just verifies the presence of the necessary dev tools.